gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/UrlWhitelistConfiguration.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa_cse.app.configurations;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Amira Kahya {@literal <amira.kahya at rte-france.com>}
+ */
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties("cse-cc")
```

Comment:
To reduce configuration classes and to align with gridcapa projects you can use general configuration properties for all cse-cc properties. Like that but then you don't retrieve only whitelist.

Reply:
The same here not all gridcapa projects use general configuration properties. So, we can discuss with the  team to harmonize in all projects.